pkgs/applications/science/electronics/kicad/base.nix

```diff
@@ -46,29 +49,22 @@ assert ngspiceSupport -> libngspice != null;
 
 with lib;
 let
-  versionConfig = versions.${baseName};
-
   # oce on aarch64 fails a test
   withOCE = oceSupport && !stdenv.isAarch64;
   withOCC = (withOCCT && !withOCE) || (oceSupport && stdenv.isAarch64);
 
-  libraries = callPackages ./libraries.nix versionConfig.libVersion;
-
 in
 stdenv.mkDerivation rec {
 
-  i18n = libraries.i18n;
+  i18n = callPackage ./i18n.nix {
+    src = i18nSrc;
+    version = i18nVersion;
+  };
 
   pname = "kicad-base";
-  version = "${builtins.substring 0 10 versions.${baseName}.kicadVersion.src.rev}";
-
-  src = fetchFromGitLab (
-    {
-      group = "kicad";
-      owner = "code";
-      repo = "kicad";
-    } // versionConfig.kicadVersion.src
-  );
+  version = kicadVersion;
```

Comment:
Would you mind clarifying this point? I'm not sure I completely understand...

Reply:
[I said before](https://github.com/NixOS/nixpkgs/pull/100066#issuecomment-706604693) that this PR triggers a change in the inputs of `kicad-unstable`, and it shouldn't. I said it shouldn't because it's only a change in the way the derivations are evaluated, and not in the derivations themselves. But, this is an example of a _justified_ "change" in one of the dependencies that triggers a rebuild, but isn't much of a big deal (only a change in the value of `version` which affects it's `name`). Hence I take back my words that this PR shouldn't trigger rebuilds - so there's nothing for you to do around this.